components/prism-core.js

```diff
@@ -80,6 +80,7 @@ var _ = _self.Prism = {
 					return clone;
 			}
 
+			// default:
```

Comment:
There seems to be some problem here.

Reply:
That shouldn’t be an issue, I just put that there so that I would remind myself that it’s only run when the above `switch` doesn’t match anything.

Reply:
Wouldn't it be a good idea to make a `default` case then?